### ISSUE TITLE
VBA: Perform Foundational testing of the FE for Staging Review

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-cms#16369]
## Description or Additional Context
We need to perform a11y testing 5 days before staging review.
- Foundational accessibility testing is _required_ before staging review. This includes
  - Use of color and color contrast
  - Automated testing with axe by Deque
  - Content zoom and reflow
  - Keyboard navigation
- Advanced accessibility testing is _recommended_ before staging review.

## Reference
- [Accessibility testing artifact template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=briandeconinck&labels=a11y-testing&template=a11y-testing.yaml&title=Accessibility+Testing+for+%5BTeam+Name%2C+Product+Name%2C+Feature+Name%5D)
- For more information on this artifact and the accessibility testing expected before your staging review, see the [how to prepare for an Accessibility Staging review guidance](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/prepare-for-an-accessibility-staging-review)

## Acceptance Criteria
- [x] A Tugboat VBA is created with facility services, to be used for both Foundational testing, and Staging Review
- [x] A11y testing is complete, including (at a minimum)
  - [x] Use of color and color contrast
  - [x] Automated testing with axe by Deque
  - [x] Content zoom and reflow
  - [x] Keyboard navigation 
- [x] Results are documented via [Accessibility testing artifact template](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=briandeconinck&labels=a11y-testing&template=a11y-testing.yaml&title=Accessibility+Testing+for+%5BTeam+Name%2C+Product+Name%2C+Feature+Name%5D)
    - [x] Any known a11y issues that won't complete for Staging Review are noted in Additional Findings field
- [x] Defects are documented in issues in backlog and flagged for prioritization
- [x] Platform cc a11y ticket is attached as artifact for Collab cycle

**Comment by eselkin** [comment]
This requires the script be created for the generation of VBA content, correct?

**Comment by davidmpickett** [comment]
Asked some qs on slack https://dsva.slack.com/archives/C0FQSS30V/p1717458219561289

**Comment by davidmpickett** [comment]
Currently blocked by Tugboat issues: https://dsva.slack.com/archives/CT4GZBM8F/p1717517716070159

**Comment by davidmpickett** [comment]
I will be using my Tugboat Demo environment for this. This morning I rebuilt the tugboat and I ran the script to add the test content and ran a content release. Hopefully these will be stable links for the duration of the next week. 
- [Philadelphia VA Regional Benefit Office](https://web-jdujzwgvp8kmj36hcfonqejlgtq5b6st.demo.cms.va.gov/philadelphia-va-regional-benefit-office/)
- [Reno VA Regional Benefit Office](https://web-jdujzwgvp8kmj36hcfonqejlgtq5b6st.demo.cms.va.gov/reno-va-regional-benefit-office/)
- [Oakland VA Regional Benefit Office](https://web-jdujzwgvp8kmj36hcfonqejlgtq5b6st.demo.cms.va.gov/oakland-va-regional-benefit-office/)

**Comment by davidmpickett** [comment]
Created the testing artifact from template: https://github.com/department-of-veterans-affairs/va.gov-team/issues/86174

**Comment by davidmpickett** [comment]
**Things to fix:**
- Broken link -> [direct deposit information online](https://web-jdujzwgvp8kmj36hcfonqejlgtq5b6st.demo.cms.va.gov/change-your-va-direct-deposit-and-contact-information)

**Comment by davidmpickett** [comment]
@omahane @eselkin 
While reviewing this, I found a tiny tech debt defect thing that is probably fine to ignore forever, but just wanted to document it for posterity https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18361

If anything, the action to take here might be updating @omahane test content csv to not have nulls for Office Visits to prevent other people from seeing this issue and flagging it.

**Comment by davidmpickett** [comment]
Tickets created:
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18367
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18354
- https://github.com/department-of-veterans-affairs/va.gov-cms/issues/18361

**Comment by davidmpickett** [comment]
@mmiddaugh @Agile6MSkinner - I have now created ticket stubs for all the issues I noticed while testing. Some of them weren't accessibility specific, so I added those to the VBA epic, but not the accessibility testing artifact. Let me know if you have any questions
